chat-plugins/wifi.js

```diff
@@ -7,6 +7,7 @@
 // checks whether any alt of the user is present in list.
 function checkAllAlts(user, list) {
 	for (var prevName in user.prevNames) {
+		if (prevName === user.userid) continue;
 		if (prevName in list) return 'previous name ' + prevName;
 	}
 	var alts = user.getAlts().map(toId);
```

Comment:
UGH

NO

STOP WASTING MY PRECIOUS CPU POWER

`.getAlts()` is _very_ CPU-intensive and only approved for bans and locks. We don't even use it for mutes.

You can record the user's IP if you want to track alts yourself.

GET RID OF IT RIGHT NOW, or I'm deleting this plugin in five days.